lib/influxdb/client.rb

```diff
@@ -149,6 +149,10 @@ def delete_shard(shard_id, server_ids)
       delete full_url("/cluster/shards/#{shard_id}"), data
     end
 
+    def write_point9(data)
+      _write9(data)
```

Comment:
well, to honor the old api, as they handle multiple points in one write, I guess it depends on where we think it's most expensive to handle multiple points...
But I'd planned to implement this , these coming days...